src/components/radio-button/radio-button.jsx

```diff
@@ -290,6 +294,24 @@ const RadioButtonComponent = (
 					{ renderLabel() }
 				</label>
 			) }
+			{ !! info && (
+				<div className="absolute mr-[2px] bottom-1.5 right-3">
```

Comment:
```suggestion
				<div className="absolute mr-0.5 bottom-1.5 right-3">
```

Reply:
It must not change need 2px always. if use rm/em it changes based on base value.

Reply:
@BabuTechBsf,  As we are using the `rem` unit all over the app if we use `px`, and if the root value gets changed then the UI might not look good. That's why IMO we should use `rem` instead.